docs/index.md

```diff
@@ -1,20 +1,22 @@
 # Welcome to the Regula Docs!
 
+**Current version: {{ version }}**
+
 [Regula](https://github.com/fugue/regula) is a tool that evaluates CloudFormation and Terraform infrastructure-as-code for potential AWS, Azure, and Google Cloud security and compliance violations prior to deployment.
 
+Regula supports the following file types:
+
+- CloudFormation JSON/YAML templates
+- Terraform HCL code
+- JSON-formatted Terraform plans
+
 Regula includes a library of rules written in Rego, the policy language used by the [Open Policy Agent](https://www.openpolicyagent.org/) (OPA) project. Regula works with your favorite CI/CD tools such as Jenkins, Circle CI, and AWS CodePipeline; we’ve included a [GitHub Actions example](https://github.com/fugue/regula-action) so you can get started quickly. Where relevant, we’ve mapped Regula policies to the CIS AWS, Azure, and Google Cloud Foundations Benchmarks so you can assess compliance posture. Regula is maintained by engineers at [Fugue](https://fugue.co).
 
 Regula is also available as a Docker image on DockerHub [here](https://hub.docker.com/r/fugue/regula).
 
 ## How does Regula work?
 
-There are two parts to Regula. The first is a [shell script](https://github.com/fugue/regula/blob/master/bin/regula) that generates a JSON document for [opa](https://www.openpolicyagent.org/) consumption.
-
-The second part is a Rego framework that:
-
--   Merges resource info from `planned_values` and `configuration` in the Terraform plan into a more conveniently accessible format, and walks through the imported Terraform modules and merges them into a flat format.
--   Looks for [rules](https://github.com/fugue/regula/tree/master/rules) and executes them.
--   Generates a report with the results of all relevant rules and [control mappings](report.md#compliance-controls-vs-rules).
+Regula's CLI tool uses an embedded [OPA](https://www.openpolicyagent.org/) preloaded with the Regula [rule library](https://github.com/fugue/regula/tree/master/rego/rules). The Regula CLI is built using Go, which means it is installed as a single self-contained binary. It reads in CloudFormation templates, Terraform HCL code, and JSON-formatted Terraform plans and evaluates them using Regula's library of rules, generating a [report](report.md). Regula also supports [custom rules](development/writing-rules.md) written in Rego. For usage details, see [Usage](usage.md).
```

Comment:
@becki-at-luminal the first sentence seems a bit awkward - is it necessary? We could just reference the library later in the paragraph when we talk about using Regula's library of rules.